models/README.md

```diff
@@ -0,0 +1,4 @@
+#<h1>
```

Comment:
Can't comment on it directly because it's too big, but do we need to check in the 28K-line `OBI_MaterialEntity_PlannedProcess_InfoContentEntity.xml.rdf` file?

Reply:
Yes, this is an ontology subset that we include.  I'll check to see if there are others we should be including